### PR TITLE
Added support for 3D batchnorm module

### LINKED ISSUE #353: Add support for nn_batch_norm3d
Hi @dfalbel,

Would you be interested in a simple PR to add support for 3D batchnorm, in line with https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm3d.html#torch.nn.BatchNorm3d?

### PR DESCRIPTION
**Pull request by mattwarkentin**
Closes #353

### CODE REVIEW

**Comment by mattwarkentin** [comment]
Oops, sorry I forgot about using the custom documentation script.

**Comment by dfalbel** [comment]
No worries! Thanks very much @mattwarkentin
